### PR TITLE
MAUI - 891167 - Provided UG contents for new features in volume 2 2024 - Development

### PR DESCRIPTION
**Pull request by PriyadharshniM**
Hi team,

I have added the UG contents for new features in .NET MAUI controls. Kindly review and merge the same.

Regards,
Priyadharshni M

### CODE REVIEW

File: MAUI/TextInputLayout/Supported-input-views.md [hunk 1]
```diff
@@ -111,12 +118,71 @@ inputLayout.Content = autocomplete;
...
 
 ## Combo box
 
 To initialize the [ComboBox](https://help.syncfusion.com/maui/combobox/overview) control and launch it in each platform, refer to the [getting started with combo box](https://help.syncfusion.com/maui/combobox/getting-started) documentation.
 
+### Selection Mode
+
+There are two different selection modes in [ComboBox](https://help.syncfusion.com/maui/combobox/selection) control.
+
+#### [Single Selection](https://help.syncfusion.com/maui/combobox/selection#single-selection)
```
Comment (Choza-rajan) [inline MAUI/TextInputLayout/Supported-input-views.md:184]: remove the link

File: MAUI/TextInputLayout/Supported-input-views.md [hunk 2]
```diff
@@ -154,7 +220,60 @@ inputLayout.Content = combobox;
 
 {% endtabs %}
 
-![Combobox](images/SupportedInputViews/ComboBox.jpg)
+![Combobox Single Selection](images/SupportedInputViews/ComboBox.jpg)
+
+#### [Multi Selection](https://help.syncfusion.com/maui/combobox/selection#multiple-selection)
```
Comment (Choza-rajan) [inline MAUI/TextInputLayout/Supported-input-views.md:225]: remove the link

File: MAUI/TextInputLayout/Supported-input-views.md [hunk 3]
```diff
@@ -74,6 +74,13 @@ inputLayout.Content = new Editor();
 
 To initialize the [Autocomplete](https://help.syncfusion.com/maui/autocomplete/overview) control and launch it in each platform, refer to the [getting started with autocomplete](https://help.syncfusion.com/maui/autocomplete/getting-started) documentation.
 
+
+### Selection Mode
+
+There are two different selection modes in [Autocomplete](https://help.syncfusion.com/maui/autocomplete/selection) control.
+
+#### [Single Selection](https://help.syncfusion.com/maui/autocomplete/selection#single-selection)
```
Comment (Choza-rajan) [inline MAUI/TextInputLayout/Supported-input-views.md:82]: remove the link

File: MAUI/TextInputLayout/Supported-input-views.md [hunk 4]
```diff
@@ -111,12 +118,71 @@ inputLayout.Content = autocomplete;
 
 {% endtabs %}
 
-![Autocomplete](images/SupportedInputViews/Autocomplete.jpg)
+![Autocomplete Single Selection](images/SupportedInputViews/Autocomplete.jpg)
+
+#### [Multi Selection](https://help.syncfusion.com/maui/autocomplete/selection#multiple-selection)
```
Comment (Choza-rajan) [inline MAUI/TextInputLayout/Supported-input-views.md:123]: remove the link

File: MAUI/ComboBox/UI-Customization.md [hunk 5]
```diff
@@ -627,9 +627,9 @@ comboBox.ItemPadding = new Thickness(10,20,0,0);
 
 ![.NET MAUI ComboBox Itempadding.](Images/UICustomization/Itempadding.png)
 
-### Customize in DropDown Width
+### Customize the DropDown Width
 
-The `DropdownWidth` property is used to modify the height of the dropdown items.
+The `DropdownWidth` property is used to modify the height of the dropdown.
```
[the remark this answers is not in the document]
Comment (PriyadharshniM) [reply MAUI/ComboBox/UI-Customization.md]: Changes updated

File: MAUI/TextInputLayout/Supported-input-views.md (hunk 1 above)
Comment (PriyadharshniM) [reply MAUI/TextInputLayout/Supported-input-views.md]: Link removed

File: MAUI/TextInputLayout/Supported-input-views.md (hunk 2 above)
Comment (PriyadharshniM) [reply MAUI/TextInputLayout/Supported-input-views.md]: Link removed

File: MAUI/TextInputLayout/Supported-input-views.md (hunk 3 above)
Comment (PriyadharshniM) [reply MAUI/TextInputLayout/Supported-input-views.md]: Link removed

File: MAUI/TextInputLayout/Supported-input-views.md (hunk 4 above)
Comment (PriyadharshniM) [reply MAUI/TextInputLayout/Supported-input-views.md]: Link removed

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**